core/serv/src/main.rs

```diff
@@ -24,6 +24,10 @@ use ya_activity::service::Activity as ActivityService;
 use ya_identity::service::Identity as IdentityService;
 use ya_net::Net as NetService;
 use ya_payment::PaymentService;
+
+#[cfg(feature = "dummy-driver")]
```

Comment:
This approach makes it only possible to import one driver at the time, think we need to fix that enum to import both.

Reply:
I'm working on it but it's difficult.

Reply:
Not really worth the pain, let's leave it like this.